client/utils/events.ts

```diff
@@ -80,6 +80,10 @@ function isEventSameDay(startingDate: IDateTime, endingDate: IDateTime): boolean
     return moment(startingDate).format('DD/MM/YYYY') === moment(endingDate).format('DD/MM/YYYY');
 }
 
+function showEventStartDate(eventDate: IDateTime, planningDate: IDateTime): boolean {
```

Comment:
Since up in the code you're passing a possibly `undefined` planningDate it only makes sense to make this optional here, and handle the case where `planningDate` is undefined explicitly